pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
```

Comment:
i think that's against the Prometheus best practices though and would require updating Prometheus config each time a new resource gets added.  As discussed here, "*A given job should be hitting a given target once with one metrics path.*"
https://github.com/prometheus/prometheus/issues/1852#issuecomment-235879630